BackEnd/src/main/resources/openapi.yaml

```diff
@@ -137,70 +150,281 @@ paths:
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/EducationEntry'
+        '500':
+          description: Internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: "Error retrieving education entries"
+        '302':
+          description: Redirect to login on expired or missing session
+          headers:
+            Location:
+              description: Login URL
+              schema:
+                type: string
+          content: { }
+    delete:
+      summary: Delete education entry
+      description: Deletes an education entry from the user's profile
+      operationId: deleteEducation
+      security:
+        - googleOAuth: [ ]
       parameters:
-        - name: jobDescriptionLink
-          in: query
-          description: URL link to the job description to analyze
-          required: true
-          schema:
-            type: string
-            format: uri
-            example: "https://example.com/job-posting"
-        - name: profileId
+        - name: educationId
           in: query
-          description: ID of the user profile
+          description: ID of the education entry to delete
           required: true
           schema:
             type: integer
             example: 1
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: "Education entry deleted"
+        '400':
+          description: Missing required parameter
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: "Missing education id parameter"
+        '500':
+          description: Internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: "Error deleting education entry"
+        '302':
+          description: Redirect to login on expired or missing session
+          headers:
+            Location:
+              description: Login URL
+              schema:
+                type: string
+          content: { }
+  /features/v0/profile/experience:
+    post:
+      summary: Add experience entry
+      description: Adds a new experience entry to the user's profile
+      operationId: addExperience
+      security:
+        - googleOAuth: [ ]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ExperienceEntry'
       responses:
         '200':
           description: Successful operation
           content:
             application/json:
               schema:
```

Comment:
Just as with education, post on experience actually returns the same value but with an id